cockatrice/src/messagelogwidget.cpp

```diff
@@ -626,20 +627,20 @@ void MessageLogWidget::logSetCardCounter(Player *player, QString cardName, int c
     QString finalStr;
     int delta = abs(oldValue - value);
     if (value > oldValue)
-        finalStr = tr("%1 places %2 %3 counter(s) on %4 (now %5).");
+        finalStr = tr("%1 places %2 %3 on %4 (now %5).");
     else
-        finalStr = tr("%1 removes %2 %3 counter(s) from %4 (now %5).");
+        finalStr = tr("%1 removes %2 %3 from %4 (now %5).");
 
     QString colorStr;
     switch (counterId) {
         case 0:
-            colorStr = tr("red", "", delta);
+            colorStr = tr("red counter(s)", "", delta);
```

Comment:
I guess this has been moved here because in some languages the adjective  also needs the singular/plural form (eg. un segnalino rosso / due segnalini rossi), am I correct?

Reply:
yes. also i'm italian and i wanted to talk with you about the italian translation since these days i've changed a lot of stuff (mostly to use official terms and whatnot). i tried using the suggestion feature on transifex but it would take forever, so i thought i'd just send you the ts file i'm working on

Reply:
Right, since @ctrlaltca reviewed most strings they are fixed and normal translators can only send suggestions. Are there that many changes required?
I personally never used the suggest feature and don't know how cumbersome it is to use...

Reply:
>